examples/vision/bit.py

```diff
@@ -0,0 +1,255 @@
+    .batch(BATCH_SIZE)
+    .prefetch(AUTO)
+)
+
+"""
+## Visualise the training samples
+"""
+
+image_batch, label_batch = next(iter(pipeline_train))
+
+plt.figure(figsize=(10, 10))
+for n in range(25):
+    ax = plt.subplot(5, 5, n + 1)
+    plt.imshow(image_batch[n])
+    plt.title(label_batch[n].numpy())
+    plt.axis("off")
+
+"""
+## Load model into KerasLayer
+"""
+
+bit_model_url = "https://tfhub.dev/google/bit/m-r50x1/1"
+bit_module = hub.KerasLayer(bit_model_url, trainable=True)
+
+"""
+## BigTransfer(BiT) model
+"""
+
+
+class MyBiTModel(keras.Model):
+    def __init__(self, num_classes, module):
+        super().__init__()
+
+        self.num_classes = num_classes
+        self.head = keras.layers.Dense(num_classes, kernel_initializer="zeros")
+        self.bit_model = module
+
+    def call(self, images):
+        # No need to cut head off since we are using feature extractor model
+        bit_embedding = self.bit_model(images)
+        return self.head(bit_embedding)
+
+
+model = MyBiTModel(num_classes=NUM_CLASSES, module=bit_module)
+
+"""
+## Define optimizer and loss
+"""
+
+learning_rate = 0.003 * BATCH_SIZE / 512
+
+# Decay learning rate by a factor of 10 at SCHEDULE_BOUNDARIES.
+lr_schedule = keras.optimizers.schedules.PiecewiseConstantDecay(
+    boundaries=SCHEDULE_BOUNDARIES,
+    values=[
+        learning_rate,
+        learning_rate * 0.1,
+        learning_rate * 0.01,
+        learning_rate * 0.001,
+    ],
+)
+optimizer = keras.optimizers.SGD(learning_rate=lr_schedule, momentum=0.9)
+
+loss_fn = keras.losses.SparseCategoricalCrossentropy(from_logits=True)
+
+"""
+## Compile the model
+"""
+
+model.compile(optimizer=optimizer, loss=loss_fn, metrics=["accuracy"])
+
+"""
+## Setting the callback
+"""
+
+train_callbacks = [
+    keras.callbacks.EarlyStopping(
+        monitor="val_accuracy", patience=2, restore_best_weights=True
+    )
+]
+
+"""
+## Train the model
+"""
+
+history = model.fit(
+    pipeline_train,
+    batch_size=BATCH_SIZE,
+    epochs=int(SCHEDULE_LENGTH / STEPS_PER_EPOCH),
+    steps_per_epoch=STEPS_PER_EPOCH,
+    validation_data=pipeline_validation,
+    callbacks=train_callbacks,
+)
+
+"""
+## Plot the model
+"""
+
+
+def plot_hist(hist):
+    plt.plot(hist.history["accuracy"])
+    plt.plot(hist.history["val_accuracy"])
+    plt.plot(hist.history["loss"])
+    plt.plot(hist.history["val_loss"])
+    plt.title("Training Progress")
+    plt.ylabel("Accuracy/Loss")
+    plt.xlabel("Epochs")
+    plt.legend(["train_acc", "val_acc", "train_loss", "val_loss"], loc="upper left")
+    plt.show()
+
+
+plot_hist(history)
+
+"""
+## Evaluate the Model
+"""
+
+accuracy = model.evaluate(pipeline_validation)[1] * 100
+print("Accuracy: {:.2f}%".format(accuracy))
+
+"""
+## Note:
+
+In this example, we trained our model for 5 epochs. In our experiment, the BigTransfer(BiT) Model performs amazing 
+by giving us a good validation accuracy. BiT performs well across a surprisingly wide range of 
```

Comment:
Avoid subjective value judgements and prefer quantified statements. Is the "experiment" here referring to training on 5 epochs, or is it about training for longer?